CODEOWNERS

```diff
@@ -0,0 +1,115 @@
+/task/buildah-24gb                          @konflux-ci/build-maintainers
+/task/buildah-6gb                           @konflux-ci/build-maintainers
+/task/buildah-8gb                           @konflux-ci/build-maintainers
+/task/buildah-min                           @konflux-ci/build-maintainers
+/task/buildah-oci-ta                        @konflux-ci/build-maintainers
+/task/buildah-remote                        @konflux-ci/build-maintainers
+/task/buildah-remote-oci-ta                 @konflux-ci/build-maintainers
+/task/buildah-rhtap                         @konflux-ci/build-maintainers
+/task/download-sbom-from-url-in-attestation @konflux-ci/build-maintainers
+/task/gather-deploy-images                  @konflux-ci/build-maintainers
+/task/git-clone                             @konflux-ci/build-maintainers
+/task/git-clone-oci-ta                      @konflux-ci/build-maintainers
+/task/init                                  @konflux-ci/build-maintainers
+/task/push-dockerfile                       @konflux-ci/build-maintainers
+/task/push-dockerfile-oci-ta                @konflux-ci/build-maintainers
+/task/show-sbom                             @konflux-ci/build-maintainers
+/task/show-sbom-rhdh                        @konflux-ci/build-maintainers
+/task/slack-webhook-notification            @konflux-ci/build-maintainers
+/task/source-build                          @konflux-ci/build-maintainers
+/task/source-build-oci-ta                   @konflux-ci/build-maintainers
+/task/summary                               @konflux-ci/build-maintainers
+/task/update-deployment                     @konflux-ci/build-maintainers
+/task/update-infra-deployments              @konflux-ci/build-maintainers
+/task/upload-sbom-to-trustification         @konflux-ci/build-maintainers
+
+# renovate groupName=build
+/task/prefetch-dependencies         @konflux-ci/build-maintainers @brunoapimentel @eskultety @taylormadore
+/task/prefetch-dependencies-oci-ta  @konflux-ci/build-maintainers @brunoapimentel @eskultety @taylormadore
+
+# renovate groupName=build
+/task/generate-labels   @konflux-ci/build-maintainers @ralphbean
+
+# renovate groupName=ec
+/task/tkn-bundle                    @konflux-ci/ec
+/task/tkn-bundle-oci-ta             @konflux-ci/ec
+/task/verify-enterprise-contract    @konflux-ci/ec
+
+# renovate groupName=integration
+/task/clair-scan                @konflux-ci/integration-service-maintainers
+/task/clamav-scan               @konflux-ci/integration-service-maintainers
+/task/deprecated-image-check    @konflux-ci/integration-service-maintainers
+/task/fbc-related-image-check   @konflux-ci/integration-service-maintainers
+/task/fbc-validation            @konflux-ci/integration-service-maintainers
+/task/inspect-image             @konflux-ci/integration-service-maintainers
+/task/sast-snyk-check-oci-ta    @konflux-ci/integration-service-maintainers
+/task/sast-unicode-check-oci-ta @konflux-ci/integration-service-maintainers
```

Comment:
Both of these tasks should also have @kdudka as a maintainer

Reply:
Makes sense, added